packages/cyberchef.vm/tools/chocolateyinstall.ps1

```diff
@@ -18,6 +18,9 @@ try {
   Install-ChocolateyZipPackage @packageArgs
   VM-Assert-Path $toolDir
 
+  # FLARE-VM adds CyberChef to the taskbar.
+  # We use the chrome executable as e can't use an `.html` shortcut for the taskbar.
```

Comment:
Minor typo: `we` is missing the `w` for `as we can't use an`

Then good to go. :)